spec/openapi.yaml

```diff
@@ -3950,6 +3997,297 @@ components:
+        gateway: "basic"
+        method: "pickup"
+        gatewayReference: "8c1ed7ea-c19a-4137-8517-7d3f580b3b97"
+        price: "4.99"
+        currency: "EUR"
+        storeId: "494c24a3-2243-484a-a81e-b3098d4dc870"
+        reference: "33eea580-2381-44bf-b355-d99a13d4f770"
+        isTest: true
+        createdAt: "2021-06-29T14:41:26+02:00"
+        updatedAt: "2021-06-29T14:41:26+02:00"
+      properties:
+        id:
+          description: Unique identifier
+          type: string
+          format: uuid
+        gateway:
+          $ref: "#/components/schemas/PaymentGatewayList"
+        method:
+          description: Payment method
+          type: string
+        gatewayReference:
+          description: Gateway reference
+          type: string
+        price:
+          description: Price in minor units
+          type: integer
+        currency:
+          description: Currency (ISO 4217 code)
+          type: string
+        storeId:
+          description: Store unique identifier
+          type: string
+          format: uuid
+        reference:
+          description: Unique reference
+          type: string
+          format: uuid
+        referrerUrl:
+          description: The URL where the payment is started from
+          type: string
+        isTest:
+          description: Is a test payment
+          type: boolean
+        createdAt:
+          description: Creation date of payment
+          type: string
+          format: date-time
+        updatedAt:
+          description: last updated date of payment
+          type: string
+          format: date-time
+          nullable: true
+        mutations:
+          description: Payment mutations
+          type: array
+          items:
+            $ref: '#/components/schemas/PaymentMutation'
+        order:
+          description: Payment order details
+          type: object
+          properties:
+            id:
+              description: Unique identifier
+              type: string
+              format: uuid
+            reference:
+              description: The order number
+              type: string
+            price:
+              description: Price  minor units
+              type: integer
+            tax:
+              description: Tax price in minor units
+              type: integer
+            locale:
+              description: Chosen currency format locale (ISO 3166-1 alpha-2 code)
+              type: string
+            deliveryType:
+              description: Delivery type
+              type: string
+              enum:
+                - "pickup"
+                - "delivery"
+            lines:
+              type: array
+              items:
+                properties:
+                  id:
+                    description: Unique identifier
+                    type: string
+                    format: uuid
+                  type:
+                    description: Line type
+                    type: string
```

Comment:
Line type can be adjusted so that it becomes an enumeration of the following types:
```
    private const DIGITAL = 'digital';
    private const DISCOUNT = 'discount';
    private const GIFT_CARD = 'gift_card';
    private const PAYMENT_FEE = 'payment_fee';
    private const PHYSICAL = 'physical';
    private const SALES_TAX = 'sales_tax';
    private const SHIPPING_FEE = 'shipping_fee';
    private const STORE_CREDIT = 'store_credit';
    private const SURCHARGE = 'surcharge';
```